wagtail/wagtailimages/templates/wagtailimages/multiple/add.html

```diff
@@ -22,6 +22,18 @@
                     <input id="fileupload" type="file" name="files[]" data-url="{% url 'wagtailimages:add_multiple' %}" multiple>
                 </div>
                 {% csrf_token %}
+                {% if collections %}
+                    <div class="field">
+                        <label for="id_addimage_collection">Add to collection:</label>
```

Comment:
`{% trans %}` tag added in 0a69dd88c1fd10f38480f4aa310ec988a0956dfd / 1aeb919a60916f28a524b93faeb478276dd2f445